bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/ActionImpl.java

```diff
@@ -31,23 +33,28 @@
  * @author Ana Dimova - Initial Contribution
  * @author Vasil Ilchev - Initial Contribution
  */
-public class ActionImpl extends ModuleImpl<ActionHandler>implements Action, ConnectedModule, SourceModule {
+public class ActionImpl extends Action implements ConnectedModule, SourceModule {
```

Comment:
I'm agree with this, but it is better to consult it with Hristo. May be a new issue has to be done for this.